src/main/java/frc/robot/subsystems/TopRollerSubsystem.java

```diff
@@ -36,4 +44,19 @@ public void setRollerSpeed(double speed) {
         topRoller.set(speed);
         Logger.getInstance().recordOutput(this.getClass().getSimpleName() + "/TopRollerspeed", speed);
     }
+
+    //-- PISTON --\\
+    public void togglePiston() {
+        rollerPiston.set(!rollerPiston.get());
+        Logger.getInstance().recordOutput(this.getClass().getSimpleName() + "/pistonState", rollerPiston.get());
+    }
+    
+    public boolean pistonIsOpen() {
+        return rollerPiston.get();
+    }
+
+    public void setPistonState(boolean open) {
+        rollerPiston.set(open);
+        Logger.getInstance().recordOutput(this.getClass().getSimpleName() + "/pistonState", rollerPiston.get());
+    }
```

Comment:
Can we add some comments for what this does? 

@JackLaFreniere @WasteOfTheOcean